x-pack/plugins/ml/common/constants/settings.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const FILE_DATA_VISUALIZER_MAX_FILE_SIZE = 'ml:fileDataVisualizerMaxFileSize';
```

Comment:
Not mandatory, but if you follow my other suggestions, it might be a good idea to synch here too:
```suggestion
export const FILE_DATA_VISUALIZER_MAX_FILE_SIZE = 'ml:DataVisualizerMaxFileSize';
```